This patch previously contained the pigeonhole sieve implementation from
https://github.com/dovecot/pigeonhole. It was automatically generated
from the package's git tree on build.

Now it should be generated from a fixed source tar ball, defined in the
Dockerfile.

--- /ERROR
+++ /ERROR
@@ -1,2 +3,4 @@
THIS PATCH IS A STUB ONLY
<see Dockerfile for patch creation>
